File: pascal/control_flow/if_statement/if_else_statement.pas
(*
 * Copyright (C) 2019-2026 Katarzyna Krasińska
 * Languagium - https://github.com/katheroine/languagium
 * Licensed under GPL-3.0 - see LICENSE.md
 *)

program IfElseStatement;

begin
  if (1 > 2) then
    writeln('1 > 2')
  else
    writeln('!(1 > 2)');

  if (2 > 1) then
    writeln('2 > 1')
  else
    writeln('!(2 > 1)');
end.

File: c#/control_flow/if_statement/IfElseStatement.cs
/*
 * Copyright (C) 2019-2026 Katarzyna Krasińska
 * Languagium - https://github.com/katheroine/languagium
 * Licensed under GPL-3.0 - see LICENSE.md
 */

class IfElseStatement {
  public static void Main(string[] args) {
    if (1 > 2)
      System.Console.WriteLine("1 > 2");
    else
      System.Console.WriteLine("!(1 > 2)");

    if (2 > 1)
      System.Console.WriteLine("2 > 1");
    else
      System.Console.WriteLine("!(2 > 1)");
  }
}
